src/shared/GlobalTopBanners/OktaErrorBanners/enums.ts

```diff
@@ -0,0 +1,45 @@
+const ErrorType = {
+  UnauthorizedClient: 'unauthorized_client',
+  AccessDenied: 'access_denied',
+  UnsupportedResponseType: 'unsupported_response_type',
+  UnsupportedResponseMode: 'unsupported_response_mode',
+  InvalidScope: 'invalid_scope',
+  ServerError: 'server_error',
+  TemporarilyUnavailable: 'temporarily_unavailable',
+  InvalidClient: 'invalid_client',
+  LoginRequired: 'login_required',
+  InvalidRequest: 'invalid_request',
+  UserCanceledRequest: 'user_canceled_request',
+} as const
```

Comment:
Is there a benefit to using this enum object here? We don't seem to be accessing the value outside of the object definition, could it just be simplified to one object?

Reply:
We're aligning with JavaScript's property naming conventions by receiving snake_case values and converting them to the preferred camelCase format

Reply:
I think I might not have explained myself well, but what I wanted to say is that since we are not actually accessing the enum properties and checking for individual keys inside the getErrorMessage function or anywhere else, there is no need to align this and we can just type it as:
```ts
const OktaErrorMappings = { 'server_error': "...", } as const
function getOktaErrorMessage(status){
  return OktaHTTPErrorMappings[status] ?? 'default status'
}
```
This way we can drop the enum and make keep the definition in a single type as opposed to having to define enum + record to make changes in the future.

Reply:
I understand your point; the way you listed it is more direct. However, this goes against our ESLint rules and JavaScript's preferred key structure. It will also make more sense if we ever need to access the enum keys, instead of having to refactor it later

![Screenshot 2024-09-16 at 10 21 06 AM](https://github.com/user-attachments/assets/4dadf73c-ed52-4b47-901e-540563790c0a)